parrot/settings.py

```diff
@@ -1,9 +1,11 @@
+import os
 import environ
 from pathlib import Path
 
 env = environ.Env()
 
 BASE_DIR = Path(__file__).parent.parent
+environ.Env.read_env(os.path.join(BASE_DIR, '.env'))
```

Comment:
Could you please use `Path`

```suggestion
environ.Env.read_env(BASE_DIR / '.env')
```

Reply:
Unfortunately, a bug in django-environ makes it impossible, see:
https://github.com/joke2k/django-environ/pull/281

I've done it another way now, without importing `os`.

Reply:
Thank you! :)